overlay_new.py

```diff
@@ -76,6 +85,11 @@ def data_field_coord(x, y):
                 self.get_data_func("reed_distance", 2, 0.001),
                 data_field_coord(3, 1),
             ),
+            DataField(
+                "VOLTAGE",
+                self.get_data_func("voltage", 1),
+                data_field_coord(3, 2),
+            ),
```

Comment:
As discussed in the meeting, it'd be great to colour this number according to battery voltage. Could you make it so that
- The voltage is red if at or below 7.00 V
- The voltage is yellow if at or below 7.30 V
- Otherwise display white, as is

Additionally, would be good to display 2 decimal places. You could consider making the text smaller to accommodate this, considering you'll be making a new component anyway.